jetty-http2/http2-server/src/main/java/org/eclipse/jetty/http2/server/AbstractHTTP2ServerConnectionFactory.java

```diff
@@ -142,7 +142,7 @@ public void setStreamIdleTimeout(long streamIdleTimeout)
 
     /**
      * @return -1
-     * @deprecated
+     * @deprecated not used anymore
```

Comment:
Typically, good `@deprecated` javadoc will include a statement indicating replacement. (even if it has "no replacement")

See: http://www.oracle.com/technetwork/java/javase/tech/index-137868.html#@deprecated

Reply:
What there is ambiguous in "not used anymore" ?

Reply:
think about the consumers of this information, those updating their usages of this class/method.
that just means it's not used here, in this class. 
was it moved somewhere else? 
has this functionality been moved (is available from) somewhere else?
is there an alternative technique?

Reply:
Wording suggestions ? "feature removed, don't use it, no replacement" ?

Reply:
+1 for that wording. its clear to me.
@WalkerWatch your thoughts?

Reply:
As Oracle suggests we should probably add, going forward, when it was deprecated. As for the wording in this instance, I like the suggestion from @sbordet. "@deprecated As of Jetty X.x.x, feature removed, no replacement."